.travis.yml

```diff
@@ -0,0 +1,29 @@
+# This config file for Travis CI
+sudo: required
+dist: trusty
+services:
+  - docker
+language: cpp
+cache: ccache
+compiler: gcc
+
+notifications:
+  email:
+    recipients:
+      - levi.armstrong@swri.org
+env:
+  global:
+    - ROS_DISTRO=kinetic
+    - UPSTREAM_WORKSPACE=file
+    - ROSINSTALL_FILENAME=dependencies.rosinstall
+    - TEST_BLACKLIST=bullet3_ros,fcl_ros,libccd_ros
+    - MAKEFLAGS="-j 4"
+    - NOT_TEST_INSTALL=true
+  matrix:
+    - ROS_DISTRO=kinetic ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO=kinetic ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO=kinetic ROS_REPO=ros TEST=clang-format
```

Comment:
Why the mixed styles? IIRC, `ROS_REPO` and `ROS_REPOSITORY_PATH` do the same if you specify the default upstream repositories.

See [this config](https://github.com/ros-industrial/ros_canopen/blob/410984dc5ec8be0e41686a42618fdfa7c037e570/.travis.yml) for an example.

Reply:
Ah, is one preferred over the other?

Reply:
I would say the "short" style is now the preferred one.

Reply:
I have added the aliases because they are more readable (in config and on travis).